.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: c
+
+os: linux
+sudo: required
+dist: trusty
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then tools/apt-get-install-deps.sh; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then tools/brew-install-deps.sh; fi
+
+install: make prerequisites
+
+script: "make -j VERBOSE=1 NINJA=1 $TARGET"
+
+env:
+  - TARGET="check-signed-off check-vera check-cpp"
+  - TARGET="build.linux test-js-precommit"
+  - TARGET=build.mcu_stm32f3
+  - TARGET=build.mcu_stm32f4
+  - TARGET=test-unit
+
+matrix:
+  include:
+    - os: osx
+      env: TARGET="build.darwin test-js-precommit"
+    - os: osx
+      env: TARGET=test-unit
+  allow_failures:
+    - os: osx
+
+#branches:
+#  only: master
```

Comment:
That's something to be decided. That's why I've left it there, to show that we have an option. But I can delete those lines ofc.